core/themes/basis/css/skin.css

```diff
@@ -400,6 +400,9 @@ fieldset .fieldset-legend {
   }
 }
 
+.l-top {
```

Comment:
The scope of this definition (gray background for the top region) is too broad. It should be restricted to the home page.

Reply:
This is fixed in the latest PR>